apps/console/src/features/applications/pages/applications.tsx

```diff
@@ -173,6 +174,7 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
             })
             .finally(() => {
                 setApplicationListRequestLoading(false);
+                setPageLoading(false);
```

Comment:
What's the difference between `isPageLoading` & `isApplicationListRequestLoading`? I feel like if we set it like this, then the state is redundant. Better if we can listen to `isApplicationListRequestLoading` and other requests in a `useEffect` and set the `isPageLoading` state and use that in all the loaders as a component wide loader state. WDYT?